src/ucs/sys/string.h

```diff
@@ -71,4 +72,15 @@ uint64_t ucs_string_to_id(const char *str);
 void ucs_memunits_to_str(size_t value, char *buf, size_t max);
 
 
+/**
+ * Extract the IP address from the given sockaddr and return it.
+ *
+ * @param sock_addr Sockaddr to take IP address from.
+ * @param ip_str    A string filled with the IP address.
+ *
+ * @return ip_str.
+ */
+char* ucs_sockaddr_str(struct sockaddr *sock_addr, char *ip_str);
```

Comment:
the parameter name should be "dest" or "str". in general, the address does not have to be an IP address.